setup.sh

```diff
@@ -123,13 +124,17 @@ if linux; then
 fi
 
 if ! hash gdb; then
-    echo 'Could not find gdb in $PATH'
+    echo "Could not find gdb in $PATH"
     exit
 fi
 
 # Update all submodules
 git submodule update --init --recursive
 
+# Set Python3 as the default.
+sudo rm -f /usr/bin/python
+sudo ln -s /usr/bin/python3 /usr/bin/python
```

Comment:
In GitHub, the default Python is re-redirect to Python2. Docker doesn't have python, it causes some crashes.

```text
[Unit tests/tests] ⭐  Run Python version info
  echo 'GDB py:'
  gdb --batch --quiet --nx --nh --ex 'py import sys; print(sys.version)'
  echo 'Installed py:'
  python -V
GDB py: 3.6.9 (default, Oct  8 2020, 12:12:24)  [GCC 8.4.0]
Installed py: Python 2.7.17
```

Reply:
Hmm I guess its just wrong to assume the installed py == gdb py here?
We should probably use `python3` invocation whenever we can, if we base upon that somehow (maybe we should not, not sure).

Reply:
> Hmm I guess its just wrong to assume the installed py == gdb py here?
> We should probably use `python3` invocation whenever we can, if we base upon that somehow (maybe we should not, not sure).

I lost the big picture and its good advice. I mean, I was involved in the code and forgot that the setup script is to install `pwndbg` in the users machines. Given this context it is not good to delete binary files. I go with your idea, check if Python3 exists, otherwise take the default.

Reply:
I'll remove these changes. The problem is only in Docker image, since we don't install `python3`, only installed `python3-dev` it's not founding the `/usr/bin/python` but in fact, in Docker all is working good and the tests passed.

Green status in `Unit test fix`: https://github.com/pwndbg/pwndbg/runs/1615157447

Summary:
```text
# -----------------------------------------------------
# Dockerfile
# -----------------------------------------------------

+ whereis python
python: /usr/bin/python3.8-config /usr/bin/python3.8 /usr/lib/python3.9 /usr/lib/python3.8 /etc/python3.8 /usr/local/lib/python3.8 /usr/include/python3.8

+ ls --color=always -lha /usr/bin/python
ls: cannot access '/usr/bin/python': No such file or directory

+ ls --color=always -lha /usr/bin/python3
lrwxrwxrwx 1 root root 9 Mar 13  2020 /usr/bin/python3 -> python3.8

++ gdb -batch -q --nx -ex 'pi import platform; print(".".join(platform.python_version_tuple()[:2]))'
+ PYVER=3.8
++ gdb -batch -q --nx -ex 'pi import sys; print(sys.executable)'
+ PYTHON+=/usr/bin/python
+ PYTHON+=3.8
PYTHON: sudo /usr/bin/python3.8


# -----------------------------------------------------
# GitHub test action
# -----------------------------------------------------

+ whereis python
python: /usr/bin/python2.7-config /usr/bin/python3.6-config /usr/bin/python /usr/bin/python2.7 /usr/bin/python3.6 /usr/bin/python3.6m /usr/bin/python3.6m-config /usr/lib/python3.7 /usr/lib/python2.7 /usr/lib/python3.8 /usr/lib/python3.6 /etc/python /etc/python2.7 /etc/python3.6 /usr/local/lib/python2.7 /usr/local/lib/python3.6 /usr/include/python2.7 /usr/include/python3.6 /usr/include/python3.6m /usr/share/python /opt/az/bin/python3.6-config /opt/az/bin/python3.6 /opt/az/bin/python3.6m /opt/az/bin/python3.6m-config /usr/share/man/man1/python.1.gz

+ ls --color=always -lha /usr/bin/python
lrwxrwxrwx 1 root root 9 Apr 16  2018 /usr/bin/python -> python2.7

+ ls --color=always -lha /usr/bin/python3
lrwxrwxrwx 1 root root 9 Oct 25  2018 /usr/bin/python3 -> python3.6

++ gdb -batch -q --nx -ex 'pi import platform; print(".".join(platform.python_version_tuple()[:2]))'
+ PYVER=3.6
++ gdb -batch -q --nx -ex 'pi import sys; print(sys.executable)'
+ PYTHON+=/usr/bin/python
+ PYTHON+=3.6
PYTHON: /usr/bin/python3.6
```

Reply:
Done in the commit da35df449f7fe98f8230d4c9eaf1d28ea6cd20cc